### PR TITLE
Update backup-all.sh

### PR DESCRIPTION
**Pull request by ghost**
remove upload to dropbox

### CODE REVIEW

File: backup-all.sh [hunk 1]
```diff
@@ -3,10 +3,11 @@
 # by piscue
 
 # Setting variables
-backup_path="/home/core/backups"
+backup_path="/opt/backups"
```
Comment (piscue) [inline backup-all.sh:6]: That could break people's path. If you want to make it generic and accept a variable I will accept the change

File: backup-all.sh [hunk 2]
```diff
@@ -3,10 +3,11 @@
...
 #backup_path="/home/core/backups"
 tar_opts="--exclude='/var/run/*'"
-cd "${BASH_SOURCE%/*}" || exit
```
Comment (piscue) [inline backup-all.sh:9]: same stuff on this path

File: backup-all.sh [hunk 3]
```diff
@@ -30,10 +31,3 @@ source backup-volumes.sh
 
 echo ""
 
-echo - upload to dropbox
```
Comment (piscue) [inline backup-all.sh:33]: The same with dropbox, if you make it optional and more generic by setting a var or argument, I'll consider it

**Comment by skrilax91** [comment]
This PR can maybe be closed

**Comment by piscue** [comment]
Stale

**Pull request closed without merging**